### PR TITLE
feature 06 pull request

### PR DESCRIPTION
**Pull request by jaredstevick**

### CODE REVIEW

**Comment by jaredstevick** [comment]
link to current WIP: https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/95

**Comment by eklemen** [comment]
:+1:

**Comment by mfees** [comment]
:+1:

**Comment by al-the-x** [comment]
You're definitely in the running for Most Entertaining Placeholder Text...

File: pub/css/main.css [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+.timeline-post:nth-child(odd) {
+    background-color: #6CBFEE;
+    border-radius: 1em;
+}
+
+.timeline-post:nth-child(even) {
+    background-color: #46A4DA;
+    border-radius: 1em;
+}
```
Comment (al-the-x) [inline pub/css/main.css:124]: Something I realize that I didn't really cover in class:

> 'Tis better to set a default than to rely on fancy selectors.

The `:nth-child()` pseudo-class is pretty well-supported at this point, but still not _fully_ supported. A browser that _doesn't_ support it (or supports it incorrectly) will ignore these rules, effectively leaving you with white text on a white background. Better to set a default color -- light blue or dark blue -- and then override that with `:nth-child()`...

**Comment by al-the-x** [comment]
FYI, on my iPad using [HTML Preview](http://htmlpreview.github.io), the time block doesn't exactly line up with the icon. See attached:

little-off-the-top

**Comment by al-the-x** [comment]
Also confirmed on Safari Desktop Mac.

File: pub/css/main.css [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    margin-bottom: 2em;
+    color: #FFFFFF;
+}
+
+.timeline time {
+    float: left;
+    display: block;
+    margin-top: -4em;
+    position: absolute;
+}
```
Comment (al-the-x) [inline pub/css/main.css:137]: Possibly the culprit...

File: pub/css/main.css [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+@media only screen and (min-width: 35em) {
+
+}
+
+@media print,
+       (-o-min-device-pixel-ratio: 5/4),
+       (-webkit-min-device-pixel-ratio: 1.25),
+       (min-resolution: 120dpi) {
+
+}
```
Comment (al-the-x) [inline pub/css/main.css:185]: Any reason not to use these?

File: pub/css/main.css [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+
+
+/* Medium Breakpoint----------- */
+@media (min-width : 768px) {
+
+header h1 {
+    font-size: 3em;
+}
```
Comment (al-the-x) [inline pub/css/main.css:193]: Do yourself a favor and indent these rules under the `@media` query to which they belong...

File: pub/css/main.css [hunk 5]
```diff
@@ -0,0 +1,371 @@
...
+time {
+    position: absolute;
+    margin-left: -8.5em;
+}
+
+
+
+
+
+
```
Comment (al-the-x) [inline pub/css/main.css:221]: So you don't end up forgetting to close the block, like you did here. Like with HTML, I generally put a comment after the closing bracket to remind me what it's supposed to close.

File: pub/index.html [hunk 6]
```diff
@@ -1,221 +1,154 @@
...
+    Changed fonts
+    Started positioning icons
+    Rounded post corners
+    Reversed icon colors
+    Changed time and date colors
+    Move time up to make room for       caret (someday)
+
+
+
+-->
```
Comment (al-the-x) [inline pub/index.html:16]: Interesting. Were these your Pomodoro notes? They'd make great commit messages... Github will even render Markdown in your extended commit messages as HTML, so you can use checklists, etc. Try it!

File: pub/index.html [hunk 7]
```diff
@@ -1,221 +1,154 @@
...
+            <h1>Vertical Timeline <small>v</small>3.0</h1>
+
+        <div class="nav">
+            <a href="#"><i class="fa fa-arrow-left"></i></a>
+            <a href="#"><i class="fa fa-arrow-right"></i></a>
+            <a href="#"><i class="fa fa-home"></i></a>
+            <a href="#"><i class="fa fa-envelope"></i></a>
+        </div>
+
+        </header>
```
Comment (al-the-x) [inline pub/index.html:58]: FYI, these headers are completely optional. I care more about the content below. Good work, though! Could you do the same with Unicode?

File: pub/index.html [hunk 8]
```diff
@@ -1,221 +1,154 @@
...
+
+
+        <!--Timeline-->
+            <ol class="timeline">
+
+        <!--Timeline Post-->
+                <li class="timeline-post">
+                    <time>
+                        <small>1/16/2015</small> <br>
+                        <big class="light">22:46</big>
```
Comment (al-the-x) [inline pub/index.html:71]: Very smart. Utility classes like this can be very handy.

**Comment by al-the-x** [comment]
Well done. Aside from the slightly offset `<time>` tags...

Flawless Victory

**Pull request merged**